Base/QTGUI/qSlicerDataDialog.cxx

```diff
@@ -94,6 +94,11 @@ qSlicerDataDialogPrivate::qSlicerDataDialogPrivate(QWidget* _parent)
   QPushButton* resetButton = this->ButtonBox->button(QDialogButtonBox::Reset);
   connect(resetButton, SIGNAL(clicked()), this, SLOT(reset()));
 
+  // do the internationalization setup for ButtonBox buttons
+  this->ButtonBox->button(QDialogButtonBox::Ok)->setText(tr("OK"));
+  this->ButtonBox->button(QDialogButtonBox::Cancel)->setText(tr("Cancel"));
+  this->ButtonBox->button(QDialogButtonBox::Reset)->setText(tr("Reset"));
```

Comment:
@u8621011 This will work for the two occurrences  found  of dialog but I think we should look for a more general approaches.

* Is there `.ts` files already available for Qt base classes ? 

* also wonder if using `QT_TRANSLATE_NOOP` as hinted [here](https://forum.qt.io/topic/47564/translate-qdialogbuttonbox/2) and [here](http://www.qtcentre.org/threads/16714-QDialogButtonBox-translation-of-standard-button-text-(-quot-OK-quot-quot-Cancel-quot-etc-)) could be more general